service/resource/deployment/resource.go

```diff
@@ -229,13 +231,15 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createState inter
 					return microerror.Mask(err)
 				}
 			case <-time.After(createTimeout):
-				return microerror.Maskf(timeoutError, "creating deployment=%#q", deploy.Name)
+				return microerror.Maskf(timeoutError, "creating Azure deployment=%#q", deploy.Name)
 			}
 
-			r.logger.Log("cluster", key.ClusterID(customObject), "debug", fmt.Sprintf("created deployment %s", deploy.Name))
+			r.logger.Log("cluster", key.ClusterID(customObject), "debug", fmt.Sprintf("creating Azure deployments: creating %#q: created", deploy.Name))
 		}
 
-		r.logger.Log("cluster", key.ClusterID(customObject), "debug", "created the deployments in the Azure API")
+		r.logger.Log("cluster", key.ClusterID(customObject), "debug", "creating Azure deployments: created")
+	} else {
+		r.logger.Log("cluster", key.ClusterID(customObject), "debug", "creating Azure deployments: already created")
 	}
```

Comment:
Besides adding logs when resource is already reconciled I made them grep friendly. Now you can do something like: `kubectl  logs ... | grep CLUSTER_ID | grep "creating Azure deployments"`. Or `kubectl  logs ... | grep CLUSTER_ID | grep "creating Azure deployments"`.

Reply:
This also applies to errors.